test/Elastic.Apm.AspNetCore.Tests/BodyCapturingTests.cs

```diff
@@ -193,7 +193,7 @@ async Task TestBodyCapture(string body, bool isError, bool shouldRequestBodyBeCa
 				var transaction = sutEnv.MockPayloadSender.FirstTransaction;
 				object capturedRequestBody = null;
 				transaction.IsSampled.Should().Be(isSampled);
-				transaction.IsContextCreated.Should().Be(isSampled, $"body: {body}");
+				//transaction.IsContextCreated.Should().Be(isSampled, $"body: {body}");
```

Comment:
I assume this is a temporary change before fixing the failing test, right?

Reply:
Oh, this was a leftover.

I'm not sure if we changed the behaviour here - Anyway, I made the test pass and uncommented the line.